src/docs/development/add-to-app/android/add-flutter-screen.md

```diff
@@ -32,8 +32,8 @@ registered in your `AndroidManifest.xml`. Add the following XML to your
 ```
 
 The reference to `@style/LaunchTheme` can be replaced by any Android theme that
-you'd like to apply to your `FlutterActivity`. The choice of theme dictates the
-colors applied to Android's system chrome, like Android's navigation bar, and 
+want to apply to your `FlutterActivity`. The choice of theme dictates the
```

Comment:
did you mean to replace it with 'that you want'?

Reply:
Yes. Sorry about dropping “you.”